os/net/bluetooth/bluetooth.c

```diff
@@ -37,12 +37,19 @@
  ****************************************************************************/
 
```

Comment:
please include <tinyara/config.h> which define tizenrt features and affect other header files